plugin/pkg/scheduler/scheduler.go

```diff
@@ -124,7 +124,9 @@ func (s *Scheduler) scheduleOne() {
 	}
 
 	go func() {
-		defer metrics.E2eSchedulingLatency.Observe(metrics.SinceInMicroseconds(start))
+		defer func() {		
```

Comment:
I don't understand this comment. I think the problem is not with start, but with that we are computing parameter of the function (so SinceInMicroseconds) here before doing binding.